FileAPI/blob/Blob-constructor.html

```diff
@@ -298,29 +315,6 @@
 });
 
 test_blob(function() {
-  return new Blob(document.createElement("div"));
-}, {
-  expected: "",
-  type: "",
-  desc: "Passing an element as the blobParts array should work."
-});
-
-test_blob(function() {
-  return new Blob(window);
-}, {
-  expected: "[object Window]",
-  type: "",
-  desc: "Passing an platform object that supports indexed properties as the blobParts array should work (window)."
```

Comment:
This is not true with window, but seems like it should be true for a nodelist, right?  Worth testing that part.